### PR TITLE
My first pull request 

### PR DESCRIPTION
**Pull request by HeDevedUp**
This simplifies the display of the current nav-bar

### CODE REVIEW

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @officailesse , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to add or improve
  - [x] Please make sure to have a professional README file. This is a general requirement. I will highly recommend following the [microverse template](https://github.com/microverseinc/readme-template)
  
  - [x] Please make sure Linter is set up correctly for this repository and all checks are green. You can find the linter template [here](https://github.com/microverseinc/linters-config)
  
  - [x] Your Pull Request should have a summary describing introduced changes in more detail (you can edit it at the top of the PR)
  - [x] Please kindly have a look at the requirements and implement the required features
  
image

     

_I'm sure you can do it just keep it up_
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -35,8 +35,8 @@ jobs:
           echo Add other actions to build,
           echo test, and deploy your project.
           
-          # run super-linter
-          - name: Super-Linter
+         
+          - name: Super-Linter,
            uses: github/super-linter@v4.9.2
```
Comment (aimemalaika) [inline .github/workflows/main.yml]: - [ ] Kindly check this link to learn how to set up HTML and  CSS linters as they are not well configured in your project [HTML&CSS linters](https://github.com/microverseinc/linters-config/tree/master/html-css).

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @officailesse, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

the-simpsons-homer-simpson

 ## Required Changes ♻️

- [x] Kindly correct the branch conflict in your repository
- [x] the file test.md should be listed in the .gitignore file so the it is not uploaded to Github as requested in the project

- [x] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HeDevedUp** [comment]
@aimemalaika 
**hello sir this is the reopened branch here  can you pls review**

File: .github/workflows/main.yml (hunk 1 above)
Comment (HeDevedUp) [reply .github/workflows/main.yml]: > @aimemalaika

pls I mistakenly merged the branch but I reopened it 
right now am done with the second project and I cant get a review. Pls can you review my work

**Pull request merged**